x/commerciokyc/types/msgs.go

```diff
@@ -190,9 +244,58 @@ func (msg MsgBuyMembership) GetSignBytes() []byte {
 
 // GetSigners Implements Msg.
 func (msg MsgBuyMembership) GetSigners() []sdk.AccAddress {
-	return []sdk.AccAddress{msg.Buyer}
+	return []sdk.AccAddress{msg.Tsp}
 }
 
+// --------------------------------
+// --- MsgRemoveMembership
+// --------------------------------
+
+type MsgRemoveMembership struct {
+	GovernmentAddress sdk.AccAddress `json:"government_address"` // Buyer address
```

Comment:
Maybe rename `GovernmentAddress` to `Government`. It's consistent with `MsgRemoveTsp`.